### ISSUE TITLE
Import order is wrong

### ISSUE THREAD

**Issue by ariel777** [krasa/EclipseCodeFormatter#107]
# Now:

```
import com.model.Ethernet;
import com.model.Id;
import com.model.Ethernet.B;
import com.model.Ethernet.PortS;
import com.model.Ethernet.PortT;

```

# Should be:
```
import com.model.Ethernet;
import com.model.Ethernet.B;
import com.model.Ethernet.PortS;
import com.model.Ethernet.PortT;
import com.model.Id;
```

# Version
Current version: 15.8.132.637.1
15.8 (2016-07-16) - Another import order fix

**Comment by krasa** [comment]
Thanks for reporting it! Please try the new version: https://github.com/krasa/EclipseCodeFormatter/releases/download/15.9.132.637.0/EclipseFormatter.zip

**Comment by krasa** [comment]
Released.

**Comment by ariel777** [comment]
Works fine now. Thank you very much for what you are doing! GBY

**Comment by drieks** [comment]
Hi @krasa,

we have the same Problem (I tested 15.9.132.637.0 and 16.0.132.637).

Given this example file (formatted by eclipse 4.4) using default import order:
```
import java.util.ArrayList;
import java.util.Map;
import java.util.Map.Entry;
import java.util.NoSuchElementException;

public class Test {
	ArrayList<String> arrayList;
	Map map;
	Entry entry;
	NoSuchElementException noSuchElementException;
}
```

EclipseCodeFormatter will always move "import java.util.Map.Entry;" one line down. (I have "Optimize imports on the fly" disabled)

When disabling "Optimize Imports" in EclipseCodeFormatter Settings (and therefore using idea optimize imports I think?) the order is correct. But then some other small changes in other files would occur.

Can you confirm this? Or is there another setting I can try?

@ariel777 could you please so kind and try to format this sample file using ours settings and tell me if the import order is correct? Do you have "Optimize Imports" enabled?

**Comment by krasa** [comment]
Ok, that's a bug.

**Comment by drieks** [comment]
Yes I can confirm that this is solved when using 4.5.1
